pkg/inserter/common_inserter.go

```diff
@@ -68,6 +68,19 @@ func (c *CommonInserter) tryParseDate(item map[string]interface{}) map[string]in
 func (c *CommonInserter) setRefs(item map[string]interface{}) map[string]interface{} {
 	for k, v := range item {
 		switch vt := v.(type) {
+		case map[string]interface{}:
+			for vtk, vtv := range vt {
+				if strings.HasPrefix(vtk, "$") {
```

Comment:
ここ、早期break?でやりたいですね
こんな感じで
```suggestion
			for vtk, vtv := range vt {
				if strings.HasPrefix(vtk, "$") {
				    continue
				}
```

Reply:
なるほどやってみる